1.4/Patches/Mods/Viles Materials.xml

```diff
@@ -299,8 +299,476 @@
 				</value>
 			</li>
 
-			</operations>
+	
```

Comment:
Im not too sure on the changes to vanilla stats. The heat Capacity in general seems to have been drastically changed